src/styles/index.scss

```diff
@@ -1,3 +1,92 @@
-body {
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+  width: 200px;
+  height: 408px;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    background-image: url(../images/imac.jpeg);
+    background-size: contain;
+    margin-top: 33px;
+  }
+
+  &__model {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), text should not be aligned to the left or right. It should be centered or justified. Please correct `text-align: left;` to `text-align: center;` if that's the design requirement.